packages/kmod-6.1-nvidia/nvidia-persistenced.service

```
@@ -0,0 +1,15 @@
+[Unit]
+Description=NVIDIA Persistence Daemon
+After=load-tesla-kernel-modules.service load-open-gpu-kernel-modules.service
+# The "nvidia" user must be created before this unit runs.
+Requires=systemd-sysusers.service
+After=systemd-sysusers.service
+
+[Service]
+Type=forking
+ExecStartPre=/bin/rm -rf /run/nvidia-persistenced
+ExecStart=/usr/libexec/nvidia/tesla/bin/nvidia-persistenced --user nvidia --verbose
+ExecStopPost=/bin/rm -rf /var/run/nvidia-persistenced
```

Better to use the same paths in both commands
```suggestion
ExecStopPost=/bin/rm -rf /run/nvidia-persistenced
```